static/src/js/util/__tests__/portals.test.js

```diff
@@ -4,6 +4,91 @@ import * as getApplicationConfig from '../../../../../sharedUtils/config'
 // eslint-disable-next-line import/no-unresolved
 import availablePortals from '../../../../../portals/availablePortals.json'
 
+jest.mock('../../../../../portals/availablePortals.json', () => ({
```

Comment:
Was using the real portals before so changes would break the test now its using a mock

Reply:
I don't understand this comment, but you've added this huge wall of text in multiple files now